### PR TITLE
README.md: rework with more information

### PR DESCRIPTION
**Pull request by jlebon**
1. link to main Fedora CoreOS website
2. no longer reference fedora-atomic repo
3. link to discussions away from repo
4. add note about repo structure

### CODE REVIEW

**Comment by dustymabe** [comment]
LGTM - cc @bgilbert

File: README.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
 Use https://github.com/coreos/coreos-assembler to build it.
+
+Discussions in
+https://discussion.fedoraproject.org/c/server/coreos and
+https://github.com/coreos/fedora-coreos-tracker.
+
+## About this repo
+
+There is one branch for each stream. The default branch is
+`testing-devel`, on which all development happens.
```
Comment (dustymabe) [inline README.md]: i wonder if this should be a link to the testing-devel branch in github.

Comment (jlebon) [reply README.md]: Updated! :arrow_down:

**Comment by sinnykumari** [comment]
LGTM

**Review by dustymabe — APPROVED** [review]

**Pull request merged**